src/components/Error.tsx

```diff
@@ -0,0 +1,20 @@
+import { ErrorType } from '../utils/ErrorType';
+
+/* eslint-disable jsx-a11y/control-has-associated-label */
+type Props = {
+  setHasError: (value: ErrorType) => void,
+  setErrorMessage: () => string,
```

Comment:
Rename 'setErrorMessage' prop to 'getErrorMessage' since it's a function that returns an error message, not sets it.